modules/single_project/main.tf

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
Since we are now referring to the upstream module, we dont need this

Reply:
@bharathkkb Remove the whole single_project directory?

Reply:
yes correct